examples/stable_diffusion_v2/train_cldm.py

```diff
@@ -300,6 +308,7 @@ def main(args):
         betas=args.betas,
         weight_decay=args.weight_decay,
         lr=lr,
+        scale_lr=args.scale_lr,
```

Comment:
A better implementation might be setting scale_lr as a float number to let users input the scaling value of lr instead of hardcoding it.

Reply:
Done. Add a new argument "group_lr_scaler" to control the scale rate.